### PR TITLE
[KeyVault] [Do Not Merge] Adding ECC key support 

### PR DESCRIPTION
**Pull request by schaabs**
## Description
These changes introduce ECC key support to the Key Vault SDK.  They correspond to the swagger update https://github.com/Azure/azure-rest-api-specs/pull/1724.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.WebKey/Microsoft.Azure.KeyVault.WebKey.csproj [hunk 1]
```diff
@@ -4,8 +4,8 @@
     <PackageId>Microsoft.Azure.KeyVault.WebKey</PackageId>
     <Description>Microsoft Azure Key Vault WebKey Class Library</Description>
     <AssemblyTitle>Microsoft Azure Key Vault WebKey</AssemblyTitle>
+    <VersionPrefix>2.0.8-alpha2</VersionPrefix>
```
Comment (shahabhijeet) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.WebKey/Microsoft.Azure.KeyVault.WebKey.csproj]: any reason you want to tag as alpha. Usually in nuget we tag such releases are preview?

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
I don't see any AssemblyInfo changes for this PR?

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -6,7 +6,7 @@
 using System.Reflection;
 using System.Resources;
 
-[assembly: AssemblyTitle("Microsoft Azure key vault")]
+[assembly: AssemblyTitle("Microsoft Azure Key Vault")]
 [assembly: AssemblyDescription("Azure Key Vault enables users to store and use cryptographic keys within the Microsoft Azure environment. Azure Key Vault supports multiple key types and algorithms and enables the use of Hardware Security Modules (HSM) for high value customer keys. In addition, Azure Key Vault allows users to securely store secrets in a Key Vault; secrets are limited size octet objects and Azure Key Vault applies no specific semantics to these objects. A Key Vault may contain a mix of keys and secrets at the same time, and access control for the two types of object is independently controlled. Users, subject to appropriate authorization, may: 1) Manage cryptographic keys using Create, Import, Update, Delete and other operations 2) Manage secrets using Get, Set, Delete and other operations 3) Use cryptographic keys with Sign/Verify, WrapKey/UnwrapKey and Encrypt/Decrypt operations. Operations against Key Vaults are authenticated and authorized using Azure Active Directory.")]
```
Comment (shahabhijeet) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault/Properties/AssemblyInfo.cs]: @schaabs any reason you do not use AssemblyFileVersion?

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Core/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 using System.Reflection;
 
-[assembly: AssemblyTitle("Microsoft Azure key vault core")]
+[assembly: AssemblyTitle("Microsoft Azure Key Vault Core")]
 [assembly: AssemblyDescription("Microsoft Azure Key Vault Core Class Library")]
```
Comment (shahabhijeet) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Core/Properties/AssemblyInfo.cs]: @schaabs any reason AssemblyFileVersion is missing.

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -5,7 +5,7 @@
 
 using System.Reflection;
 
-[assembly: AssemblyTitle("Microsoft Azure key vault cryptography")]
+[assembly: AssemblyTitle("Microsoft Azure Key Vault cryptography")]
 [assembly: AssemblyDescription("Microsoft Azure Key Vault Cryptography Class Library.")]
```
Comment (shahabhijeet) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Properties/AssemblyInfo.cs]: @schaabs any reason AssemblyFileVersion is missing

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Comment by schaabs** [comment]
closing since this will be merged into the KvDev branch instead. https://github.com/Azure/azure-sdk-for-net/pull/3815

**Pull request closed without merging**